### PR TITLE
add aikit application card, getting-started markdown

### PR DESCRIPTION
**Pull request by kkasravi**
this is similar to #33 - just adding the aikit application and quickstart cards

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
We are in the process of getting the downstream changes merged back upstream. Once done this PR should likely be added upstream.

File: data/applications/aikit.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: aikit
```
Comment (jeff-phillips-18) [inline data/applications/aikit.yaml:2]: Please add an annotation for the categories the documentation can be filtered by. ie:

`  annotations:
    opendatahub.io/categories: 'AI/Machine learning,Deployment,Model serving'`

Current categories we use are: 
AI/Machine learning,
Data management,
Data visualization,
Deployment,
Getting started,
Jupyter notebook,
Model serving,
Monitory,
Python,
SciKit-Learn,
TensorFlow,
PyTorch

File: data/applications/aikit.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  description: |-
+    A comprehensive set of interoperable AI software tools, built using oneAPI, to provide drop-in acceleration for end to end data science and machine learning pipelines on Intel architectures with familiar Python libraries and frameworks.
+  kfdefApplications: ['jupyterhub', 'notebook-images']
+  route: jupyterhub
+  img: images/oneapi.png
+  category: Intel® managed
+  support: Intel®
+  docsLink: https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html
+  quickStart: create-aikit-notebook
+  getStartedLink: https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top.html
```
Comment (jeff-phillips-18) [inline data/applications/aikit.yaml:15]: This should be behind a feature flag until it is certified:

`featureFlag: aikit`

Comment (kkasravi) [reply data/applications/aikit.yaml:17]: Done

**Review by jeff-phillips-18 — COMMENTED** [review]
Why the changes to package-lock.json files?

**Comment by kkasravi** [comment]
@jeff-phillips-18 
I redid this PR. Just 3 files that I've added. The 4th file [OdhDocCard.tsx](https://github.com/red-hat-data-services/odh-dashboard/blob/9b33ccef00780e165c14d1223189b900eae322b3/frontend/src/components/OdhDocCard.tsx) was updated because of a lint failure. I noticed that if I run 'npm run test:lint' under frontend I get a lint error asking me to undo these changes. So the PR lint tests seem to differ from the frontend lint test.

**Review by jeff-phillips-18 — APPROVED** [review]

**Comment by jeff-phillips-18** [comment]
> I noticed that if I run 'npm run test:lint' under frontend I get a lint error asking me to undo these changes. So the PR lint tests seem to differ from the frontend lint test.
OK, thanks. I'll take a look and see what's going on.

**Pull request merged**